extensions/amp-consent/0.1/test/test-consent-ui.js

```diff
@@ -526,6 +527,79 @@ describes.realWin(
             'info': 'Entering fullscreen.',
           });
         });
+
+        describe('actionPromptTrigger', () => {
+          it('should not expand when actionPromptTrigger is false', async () => {
```

Comment:
By default actionPromptTrigger would be false, this looks duplicate.

Reply:
fair point. Removing test.